app/globals.css

```
@@ -418,4 +418,41 @@ a{
+  }
+  to {
+    transform: translateY(100%);
+  }
+}
+
+.slide-down-exit {
+  animation: slideDown 0.3s ease-in forwards;
+}
+
+.nav-rail{
+  height: calc(100vh- var(--mobile-content-offset));
```

Biome (1.9.4)

[error] Unexpected unknown unit: vh-

See MDN web docs for more details.
Use a known unit instead, such as:


(lint/correctness/noUnknownUnit)